README.md

```diff
@@ -49,3 +49,24 @@ console.log(moveElementByValue(array, 'A', 'C'));  // -> ['B', 'A', 'C'] (Insert
 console.log(moveElementByValue(array, 'A', 'C', true));  // -> ['B', 'C', 'A'] (Insert 'A' after 'C')
 console.log(moveElementByValue(array, 'D', 'C'));  // -> Error! ('D' is not included)
 ```
+
+### `moveElementBy`
+
+```js
+const {moveElementBy} = require('..');
```

Comment:
Please replace `require('..');` to the following code:

```
require('sort-array-for-drag-and-drop');
```

Reply:
This `require('..');` is the same to the soruce codes in examples directory.
Should I fix these codes too ?

Reply:
I fixed this